easybuild/easyconfigs/s/Subread/Subread-1.5.0p1-foss-2016a.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+easyblock = 'MakeCp'
+
+name = 'Subread'
+version = '1.5.0-p1'
+
+homepage = 'http://subread.sourceforge.net/'
+description = """High performance read alignment, quantification and mutation discovery"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['%(namelower)s-%(version)s-source.tar.gz']
+source_urls = [('https://sourceforge.net/projects/%(namelower)s/files/%(namelower)s-%(version)s', 'download')]
+
+prebuildopts = 'cd src;'
```

Comment:
use `start_dir = 'src'`

effect should be the same though

if you prefer this, then replace `; ` with `&&` (to chain the commands together such that one subcommand failing results in cancelling the rest)